kernel-headers/rdma/mana-abi.h

```diff
@@ -16,8 +16,20 @@
 
 #define MANA_IB_UVERBS_ABI_VERSION 1
 
+enum mana_ib_create_cq_flags {
```

Comment:
Changes to this file should be created with kernel-headers/update script and be in separate commit.

Thanks

Reply:
Thanks! I did not know that.

I will do in the next PR. I abandon this PR as I will make one PR with all RC QP related changes, which will include this one.